packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -556,253 +576,262 @@ export class StatsAnalyzer extends EventsScope {
-        previousStats = this.lastStatsResults[mediaType].send;
+          currentStats = this.statsResults[mediaType].recv;
+          previousStats = this.lastStatsResults[mediaType].recv;
 
-        if (
-          currentStats.totalPacketsSent === previousStats.totalPacketsSent ||
-          currentStats.totalPacketsSent === 0
-        ) {
-          LoggerProxy.logger.info(
-            `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} RTP packets sent`
-          );
-        } else {
           if (
-            currentStats.framesEncoded === previousStats.framesEncoded ||
-            currentStats.framesEncoded === 0
+            currentStats.totalPacketsReceived === previousStats.totalPacketsReceived ||
+            currentStats.totalPacketsReceived === 0
           ) {
             LoggerProxy.logger.info(
-              `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} frames getting encoded`
+              `StatsAnalyzer:index#compareLastStatsResult --> No ${simplifiedMediaType} RTP packets received`
             );
+          } else {
+            if (
+              this.statsResults.resolutions[mediaType].recv.framesReceived ===
+                this.lastStatsResults.resolutions[mediaType].recv.framesReceived ||
+              this.statsResults.resolutions[mediaType].recv.framesReceived === 0
+            ) {
+              LoggerProxy.logger.info(
+                `StatsAnalyzer:index#compareLastStatsResult --> No ${simplifiedMediaType} frames received`
+              );
+            }
+
+            if (
+              this.statsResults[mediaType].recv.framesDecoded ===
+                this.lastStatsResults[mediaType].recv.framesDecoded ||
+              (this.statsResults.resolutions[mediaType].send &&
+                this.statsResults.resolutions[mediaType].send.framesDecoded === 0)
+            ) {
+              LoggerProxy.logger.info(
+                `StatsAnalyzer:index#compareLastStatsResult --> No ${simplifiedMediaType} frames decoded`
+              );
+            }
+
+            if (
+              this.statsResults.resolutions[mediaType].recv.framesDropped -
+                this.lastStatsResults.resolutions[mediaType].recv.framesDropped >
+              10
+            ) {
+              LoggerProxy.logger.info(
+                `StatsAnalyzer:index#compareLastStatsResult --> ${simplifiedMediaType} frames are getting dropped`
+              );
+            }
           }
 
-          if (
-            this.statsResults.resolutions[mediaType].send.framesSent ===
-              this.lastStatsResults.resolutions[mediaType].send.framesSent ||
-            this.statsResults.resolutions[mediaType].send.framesSent === 0
-          ) {
-            LoggerProxy.logger.info(
-              `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} frames sent`
-            );
-          }
+          this.emitStartStopEvents(
+            simplifiedMediaType,
```

Comment:
I'm not sure if using `simplifiedMediaType` here will work correctly - let's have a quick call to discuss this.